config/settings.yml

```diff
@@ -31,3 +31,6 @@ preservation_catalog:
 email_addresses:
   discussion_list: 'user@discussion_list.org'
   user_list: 'user@user_list.org'
+
+features:
+  transfer_bag_via_scp: false
```

Comment:
Keep current behavior (xfer via tarpipe) default. This way we can flip on this feature on a per-env basis and test it.